contracts/SignaturitUser.sol

```diff
@@ -361,4 +365,26 @@ contract SignaturitUser is SignaturitUserInterface {
 
         boolArrayAttr[bytes32key].length--;
     }
+
+    function _isManager(
```

Comment:
Is necessary think better all of this, because all te managers can modify data added by others and I not sure if this is good

Reply:
no, all the managers can modify just the user they manage, everytime an admin create an user it is automatically a manager of that user. External managers can't modify users belonging to others manager